src/Classifiers/ControllerClassifier.php

```diff
@@ -13,6 +15,21 @@ public function getName()
 
     public function satisfies(ReflectionClass $class)
     {
+        return $this->extendsBaseController($class) || $this->hasBeenRegisteredInARouter($class);
+    }
+
+    protected function extendsBaseController($class)
+    {
         return $class->isSubclassOf(\Illuminate\Routing\Controller::class);
     }
+
+    protected function hasBeenRegisteredInARouter($class)
```

Comment:
`hasBeenRegisteredInARouter` -> `hasBeenRegisteredInRouter`

Do not use the "a" article.